machineconfiguration/v1/types.go

```diff
@@ -503,8 +503,73 @@ type MachineConfigPoolStatus struct {
 	// +listType=atomic
 	// +optional
 	CertExpirys []CertExpiry `json:"certExpirys"`
+
+	// poolSynchronizersStatus is the status of the machines managed by the pool synchronizers.
+	// +openshift:enable:FeatureGate=PinnedImages
+	// +listType=map
+	// +listMapKey=name
```

Comment:
did you mean for this to be name,poolSynchronizerType

Reply:
good point, yes